src/world.rs

```diff
@@ -6,6 +6,7 @@ use std::sync::atomic::{AtomicU32, Ordering};
 
 use crate::archetype::{Archetype, Comp, CompMut, TypeMetadataSet};
 
+/// The ECS world storing [Entity] and components.
```

Comment:
I don' think Entity should be a link or proper name here, as it is a handle and not the entity itself. Maybe just "entities and their components"?

Reply:
I understand now that you did this consistently and I did warm up to it. So let's ignore my initial reservations...